### DISCUSSION TITLE
Add support for 7TV extensions

### DISCUSSION THREAD

**Discussion by Infinitay** [Chatterino/chatterino2#2827] [Ideas] [answered]
[7TV](https://7tv.app/) is yet another extension for emotes on Twitch offering a variety of emotes in different styles such as regular, animated, wide, and animated and wide. As far as I'm aware this is fairly new. They [have already forked Chatterino](https://github.com/SevenTV/chatterino7) in order to add support for their emotes. Perhaps this would make adding their emote service to Chatterino with ease compared to starting fresh.

Would be nice if @AnatoleAM could perhaps create a pull request to add support on the native Chatterino client for 7tv.

---

More information about [7TV](https://7tv.app/)'s extension and service could be found on their GitHub repo, https://github.com/SevenTV/SevenTV.

**Comment by jupjohn** [comment]
There was an issue about this recently but I can't seem to find it :/ 

I see two sides to this:
7tv is very niche and adding something just because it exists isn't exactly a good reason to spend time implementing something. 
Although since chatterino is used by a fair chunk of the Twitch community, an implementation of this would bring it into the mainstream which might be what it needs.

Would be nice to hear @fourtf's opinion on this (or any other maintainer)

**Comment by pajlada** [comment]
As there's already a fork providing this functionality I don't see a particular need to rush into this - I would personally rather take it slow and let the extension mature before considering adding it into Chatterino.

**Comment by serniebanders** [comment]
Now that 7tv emotes are supported as a Frankerfacez extension, it may be a good idea to try to add 7tv support now.

Source: https://github.com/FrankerFaceZ/Add-Ons/pull/73

**Reply by leon-richardt** [reply]
How does that change anything in regards to the reasoning given by @pajlada?

**Comment by killertofus** [comment]
relevant #3071

**Comment by Wiloden** [comment]
bump

**Reply by jupjohn** [reply]
No point bumping, see https://github.com/Chatterino/chatterino2/pull/3071

**Comment by the-tuxedo-cat** [comment]
omg bump

**Comment by Nerixyz** [comment]
A heads up: #4002 attempts to implement 7TV emotes and badges. Followup PRs will likely implement live emote(-set) updates, paints and animated profile pictures.

**Comment by Nerixyz** [comment, marked as answer]
Last bump, because it's added now: [3e41b84](https://github.com/Chatterino/chatterino2/commit/3e41b84ed7d41ca8f8ec7b87b193efb64393fd34) 🙂
